### ISSUE TITLE
Bootstrap 'back to top' buttons do not move focus

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1331]
The top buttons at the end of each section on Bootstrap pages do not move the focus back up to the top when clicked. Therefore, if you are using the keyboard only, tabbing then jumps you straight back down the page again to where you came from

**Comment by torinfo** [comment]
@FayCross For some reason your change does not work. The #skipLink anchor for some reason does not take the dynamic target of the link into account, so it always seems to navigate to the top of the first page (event though the link is correct).

I changed it a bit and even had to make sure that it was evaluated a bit later than in the normal flow (using a setTimeout).

It now seems to work as intended:
1. Navigates to top of page
2. When tabbing, starts at top of page again.

Could you please verify my change and that the change still works as you intended?

**Comment by torinfo** [comment]
@FayCross Re-opened issue for you to check.

**Comment by torinfo** [comment]
@FayCross In testing this, should this not just point to the current page instead of the first section? Now, the title of the first section is not visible. So for page 2, should the anchor just be #page2

**Comment by FayCross** [comment]
@torinfo I've changed this back to something similar to what I had before - so it jumps to skipLink (which is at the top of the page, not the 1st section) but it actually works this time. It was trying to find a page with the ID 'skipLink' and when it wasn't found it moved to page 1. Sorry for not noticing this when I committed the first attempt - I can only assume I tested on the first page of a project so it looked like it was working correctly. Feel free to reopen if you're not happy with my change and we can talk about it at the meeting later
